playwright/tests/pageTypes/pageTypeBuilder.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from "@playwright/test";
+
+import { test } from "../../fixtures";
+
+test.describe("Page types builder", () => {
+  test.run()(
+    "I can see default SEO & Metadata tab fields",
+    async ({ pageTypesBuilderPage, reusablePageType }) => {
+      await pageTypesBuilderPage.goto(reusablePageType.name);
+      await pageTypesBuilderPage.getTab("SEO & Metadata").click();
+
+      await expect(
+        pageTypesBuilderPage.getStaticZoneListItemFieldName("Meta Description"),
+      ).toBeVisible();
+      await expect(
+        pageTypesBuilderPage.getStaticZoneListItemFieldId(
+          "data.meta_description",
+        ),
+      ).toBeVisible();
+
+      await expect(
+        pageTypesBuilderPage.getStaticZoneListItemFieldName("Meta Image"),
+      ).toBeVisible();
+      await expect(
+        pageTypesBuilderPage.getStaticZoneListItemFieldId("data.meta_image"),
+      ).toBeVisible();
```

Comment:
This pattern of checking the field name and field ID occurs several times. Is it possible to make checking a field easier?

For example:
```ts
pageTypesBuilderPage.checkStaticZoneField({
  name: "Meta Image",
  id: "meta_image"
})
```

Reply:
Good one, replied on the next comment 🙂